charts/partials/templates/_proxy.tpl

```diff
@@ -168,7 +168,7 @@ readinessProbe:
     path: /ready
     port: {{.Values.proxy.ports.admin}}
   initialDelaySeconds: 2
-{{- if .Values.proxy.nativeSidecar }}
+{{- if and .Values.proxy.nativeSidecar (not .Values.proxy.component) }}
```

Comment:
This results in the control plane controllers failing to connect at the beginning. They do converge to a stable state after some time, but after multiple restarts (I observed 3 for destination). I think it's worth providing a clean start as much as possible by giving their proxies a startup probe with enough slack.

Reply:
Is the livenessProbe failing on the controller containers?  I'm having a lot of trouble reproducing this.  I restricted my test cluster's network bandwidth to 1Mb/s and the proxy pod happily sat there for almost 10 minutes waiting for the controllers to come up.

Reply:
I was artificially able to reproduce it by scaling the identity deployment to 0 during install.  It seems in the `nativeSidecar=false` mode the await postStart keeps it from crashlooping.  In hindsight, it makes sense to make the startupProbe behave the same as await postStart and to use the same flag for the conditional that enables it.

Reply:
Do you want the proxy startupProbe parameters to be user-configurable and supported in the injector?

Reply:
We can make `initialDelaySeconds`, `periodSeconds`and `successThreshold` parameters in `Values.proxy`, defaulted to the values you currently have, and overridden only for the destination and injector controllers via the `set $tree.Values.proxy` mechanism already in place for other parameters in the templates. The overriding values can be hard-coded for now, no need to expose to users.